docs/essentials/collections/threadsafe/blockingCollectionOverview.md

```diff
@@ -1 +1,115 @@
+# BlockingCollection Overview
+
+[BlockingCollection&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Concurrent.BlockingCollection%601.html) is a thread-safe collection class that provides the following features:
+
+*   An implementation of the Producer-Consumer pattern.
+
+*   Concurrent adding and taking of items from multiple threads.
```

Comment:
I don't see the connection with LINQ.  `Take()` on an `IEnumerable` doesn't modify the sequence; it generates a new sequence.  `Take()` on a `BlockingCollection<T>` removes that item from the collection (according to the MSDN reference; I haven't ever used this before).

The issue I have with stating "take" at the beginning was that, although it's the word used by the collection to mean "remove using the semantics of the underlying collection", it shows up in this bullet point as if the reader already understands that vocabulary.